apps/camera/style/controls.css

```diff
@@ -43,7 +43,7 @@
 .controls-left {
   justify-content: flex-end;
   opacity: 1;
-  transition: opacity 200ms ease-in-out;
+  transition: opacity 200ms ease-in-out, visibility 0ms ease-in-out;
```

Comment:
nit: for the 0ms `visibility` transition, its unnecessary to specify an easing function